resources/js/components/MedialibraryFormField.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <DefaultField :field="field" :errors="errors" :full-width-content="true" :show-help-text="showHelpText">
+  <DefaultField :field="field" :errors="errors" :full-width-content="fullWidthContent" :show-help-text="showHelpText" >
```

Comment:
This change is up for discussion, see [e73430d](https://github.com/dmitrybubyakin/nova-medialibrary-field/pull/193/commits/e73430d8a171459d3482748682cbaae4ad3d4975) for more details.

Reply:
Quick addition: I just noticed that this would only affect the form field. The question about whether we _want_ this change or not remains the same though.